modules/t/LDFeatureContainerAdaptor.t

```diff
@@ -17,7 +17,7 @@ use strict;
 use warnings;
 use Test::More;
 use Test::Exception;
-use Test::Warnings qw(warning);
+use Test::Warnings ':no_end_test';
```

Comment:
Should we simply remove the line instead ?

Reply:
I'm actually testing that a warning is returned from the adaptor in certain cases.